### DISCUSSION TITLE
# 【求助】苹果移动端在浏览绳网时很卡并且无法发送图片，只能以文件形式发送😭

### DISCUSSION THREAD

**Discussion by nayina1145141919810** [share121/inter-knot#1906] [崩坏：星穹铁道]
![752401](https://github.com/user-attachments/assets/53e802be-32ab-4e66-b37b-b9c68c4a7f68)

**Comment by baolovey** [comment]
挂加速器

**Reply by nayina1145141919810** [reply]
推荐一下谢谢🙏小白不知道用啥好

**Comment by zhang090613** [comment]
Shadowrocket，这个挺好用的，只不过要自己找机场

**Comment by wucaibanlandehei9** [comment]
wc，屁股肉丝2024愚人节封面

**Comment by snivm** [comment]
自用，稳定，按流量计费，不限时长，流量用完前不会重置，每月不会刷新，支持Netflix和chatgpt,最低1元套餐，mojie.me/#/register?code=XdLgjGwe​

**Comment by LveaiJ9** [comment]
我嘞个Retribution~Cycle of Redenption~
